pages/community/community-support-precice.md

```diff
@@ -0,0 +1,52 @@
+
+## Funding and support options
+
+There are three different ways to fund and support the development of preCICE.
+
+### 1. Research project partner
+
+Include preCICE as a project partner in your research project proposal or team up with the preCICE developers to write a proposal. This is the best option when you want to benefit from the research and scientific expertise of the preCICE team, and increase the chances that your project is funded. The developers of preCICE are experts in state-of-the-art numerical methods, high-performance computing, and software engineering for multi-X coupled problems. Reviewers appreciate the highly recognized expertise of preCICE, demonstrated by our fast growing [number of citations](https://scholar.google.com/scholar?hl=en&cites=5053469347483527186), and [vibrant community](community-projects.html).
+
+Directly [contact Jun. Prof. Benjamin Uekermann](https://www.ipvs.uni-stuttgart.de/departments/us3/) to discuss a collaboration.
+
+### 2. Support license
+
+A support license is the easiest way to support preCICE. By acquiring a support license, you directly contribute to the development of preCICE through a fixed amount per year. In exchange, you receive a number of benefits, in particular timely and direct support from the preCICE developers around any issues that may arise when using preCICE. We provide two service levels, with discounted rates for academia.
+
+#### Direct cost in a research project
+
+As an academic user you can declare the support license, like any software license, as a direct cost/consumable in your research proposal (see e.g. section 2.1 in [basic module DFG guidelines](https://www.dfg.de/formulare/52_01/52_01_en.pdf)). It is also recommended to include the support license as a risk mitigation strategy to highlight that you, as a project lead, have the support for a successful implementation. In addition, as part of the support license, you can leverage the preCICE website and community for your communication/dissemination activities.
+
+[Speak to Jun. Prof. Benjamin Uekermann](https://www.ipvs.uni-stuttgart.de/departments/us3/) to learn more.
+
+#### Benefits
+
+|   | Basic support license <br>(12 months, billed anually)  | Extended support license <br>(12 months, billed anually)  |
+| - | - | -|
+| **discounts for preCICE Workshop** | up to 3 tickets | up to 3 tickets |
+| **priority support**   | via Discourse, up to 3 badges | via Discourse, up to 3 badges  |
+| **visibility/dissemination**   | listed as supporter, project featured on website | listed as supporter, project featured on website  |
+| **private support**   |  | via email or Zoom |
```

Comment:
```suggestion
| **private support**   |  | via Discourse or Zoom |
```
There are also private communications on Discourse. I guess we would prefer these over email.

Reply:
I would also change "Zoom" to something like "online meetings" / "videoconference".

We should also try the private communication features of Discourse, first.